### PR TITLE
[ZEPPELIN-3283] Remove redundant method InterpreterOutput.getInterpreterResultMessages()

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
Remove Redundant method `InterpreterOutput.getInterpreterResultMessages()`

### What type of PR is it?
Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-3283

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public void testGrpcFrameSize() throws InterpreterException, IOException {
     InterpreterContext context = getInterpreterContext();
     result = interpreter.interpret("print(11111111111111111111111111111)", context);
     assertEquals(InterpreterResult.Code.ERROR, result.code());
-    List<InterpreterResultMessage> interpreterResultMessages = context.out.getInterpreterResultMessages();
+    List<InterpreterResultMessage> interpreterResultMessages = context.out.toInterpreterResultMessage();
```
Comment (masyan) [inline python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java]: maybe rename to    toInterpreterResultMessage**s**

**Comment by masyan** [comment]
LGTM

**Comment by zjffdu** [comment]
LGTM

File: python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java (hunk 1 above)
Comment (Leemoonsoo) [reply python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java]: Right.

Renaming can be handled in separate issue I think.

**Comment by Leemoonsoo** [comment]
Thanks @masyan @zjffdu @felixcheung for review. Merge to master and branch-0.8

**Pull request closed without merging**